SQLite3/module.modulemap

```diff
@@ -0,0 +1,5 @@
+module SQLite3 [system] {
+    header "/usr/include/sqlite3.h"
```

Comment:
I think it's safer to have a module.modulemap per platform, though this can become problematic when dealing with Xcode-beta.app. I know the SwiftPM team is attempting to solve the static module map issue in the future.

```
  s.ios.module_map = "Supporting Files/Module Maps/ios/module.modulemap"
  s.osx.module_map = "Supporting Files/Module Maps/osx/module.modulemap"
```